cmd/contour-operator.go

```diff
@@ -41,13 +41,17 @@ func init() {
 }
 
 var (
-	image                string
+	contourImage         string
+	envoyImage           string
 	metricsAddr          string
 	enableLeaderElection bool
 )
 
 func main() {
-	flag.StringVar(&image, "image", "docker.io/projectcontour/contour:main", "The image used for the managed Contour.")
+	flag.StringVar(&contourImage, "contour-image", "docker.io/projectcontour/contour:main",
+		"The container image used for the managed Contour.")
+	flag.StringVar(&envoyImage, "envoy-image", "docker.io/envoyproxy/envoy:v1.15.1",
```

Comment:
FYI, we've updated Contour's main branch [to use Envoy 1.16.0](https://github.com/projectcontour/contour/pull/3029).